imports/ui/views/positions/Position/PositionContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import { Meteor } from 'meteor/meteor'
+import { createContainer } from 'meteor/react-meteor-data'
+import React, { Component } from 'react'
+
+import { Positions } from '/imports/api/Positions'
+import { Organizations } from '/imports/api/Organizations'
+import { Tags } from '/imports/api/Tags'
+import Position from './Position'
+
+class PositionContainer extends Component {
+
+  render() {
+    return <Position { ...this.props } />
+  }
+
+}
+
+PositionContainer.propTypes = {
+
+}
+
+export default createContainer((props) => {
+  const positionQuery = { _id: props.position._id}
+  const positionSubscription = Meteor.subscribe('Positions.get', positionQuery)
+  if (!positionSubscription.ready()) return { loading: true, position: {}, organization: {} }
+  let position = Positions.findOne(positionQuery)
+
+  const orgQuery = { positions: props.position._id }
+  const tagQuery = {_id: {$in: position.skills}}
+  const subs = [
+    Meteor.subscribe('Organizations.get', orgQuery),
+    Meteor.subscribe('Tags.get', tagQuery)
+  ]
+
+  if (_.some(subs, (s) => !s.ready())) return { loading: true, position: {}, organization: {} }
+  let organization = !props.organization._id ? Organizations.find(orgQuery).fetch()[0] : props.organization
```

Comment:
Are we expecting this ternary statement to trigger if `props.organization` isn't provided? If so, trying to grab `props.organization._id` will throw an error.